test/hcl.go

```diff
@@ -106,6 +106,12 @@ func assertTerramateRunBlock(t *testing.T, got, want *hcl.RunConfig) {
 		return
 	}
 
+	if want.CheckGenCode != got.CheckGenCode {
```

Comment:
now we can do:

```
assert.IsTrue(t, want.CheckGenCode == got.CheckGenCode, 
    "want.Run.CheckGenCode %v != got.Run.CheckGenCode %v", want.CheckGenCode, got.CheckGenCode)
```
but just an idea... feel free to ignore.

Reply:
Forgot we now have this, wiill do, thanks :heart:

Reply:
I even looked for it but I used go doc and I think go doc showed me the version on the go.mod, which doesn't have this, so I thought I was just going crazy :laughing: